library/spdm_requester_lib/libspdm_req_key_exchange.c

```diff
@@ -468,8 +469,11 @@ static libspdm_return_t libspdm_try_send_receive_key_exchange(
     if (heartbeat_period != NULL) {
         *heartbeat_period = spdm_response->header.param1;
     }
-    *req_slot_id_param = spdm_response->req_slot_id_param;
-    if (spdm_response->mut_auth_requested != 0) {
+
+    *req_slot_id_param = spdm_response->req_slot_id_param & 0xf;
+    mut_auth_requested = spdm_response->mut_auth_requested & 0xf;
```

Comment:
Should it be `& 0x7` ? Only BIT0, BIT1, BIT2 are valid.

Reply:
Indeed. Good catch.